test/diff.jl

```diff
@@ -107,3 +111,76 @@
         @test fdiff(A, dims=1) == fdiff(float.(A), dims=1)
     end
 end
+
+@testset "fgradient" begin
+    for T in generate_test_types([N0f8, Float32], [Gray, RGB])
```

Comment:
```suggestion
    for T in generate_test_types(Any[N0f8, Float32], Any[Gray, RGB])
```
(tiny bit easier on compiler and speeds up the tests)

Reply:
I'll fix this and elsewhere in a separate PR.

Reply:
That's fine. If you're fixing inference triggers in SnoopCompile these may show up without the `Any`s but that's just "analysis noise" and not anything concerning.